components/layout_2020/display_list/mod.rs

```diff
@@ -675,10 +675,10 @@ impl<'a> BuilderForBoxFragment<'a> {
     fn build_border(&mut self, builder: &mut DisplayListBuilder) {
         let border = self.fragment.style.get_border();
         let widths = SideOffsets2D::new(
-            border.border_top_width.px(),
-            border.border_right_width.px(),
-            border.border_bottom_width.px(),
-            border.border_left_width.px(),
+            border.border_top_width.0.round_to_pixels().px(),
+            border.border_right_width.0.round_to_pixels().px(),
+            border.border_bottom_width.0.round_to_pixels().px(),
+            border.border_left_width.0.round_to_pixels().px(),
```

Comment:
I think this might be incorrect as the "pixels" used during layout are not device pixels. A hidpi scale is applied in WebRender. This scaling should move back into layout pretty soon, I think which will make it easier to round to device pixels there.